EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+title: Predictable Proof-of-Work (POW) Sunsetting 
+author: @Query0x
+discussions-to: <tbd>
```

Comment:
This field is necessary.  I recommend creating a thread at https://ethereum-magicians.org/.  If you are unable or unwilling to do that, then a new issue in this repository for discussion is an acceptable (but not preferred) alternative.

Reply:
Tried signing up there but can't get past the confirmation email... requested it be resent three times.  Not in junk or spam.

Reply:
Done... what's one more email address